package.json

```diff
@@ -134,55 +114,76 @@
     ]
   },
   "scripts": {
+    "precommit": "pretty-quick --staged",
+    "prepush": "tsc -p browser/tsconfig.json && yarn lint",
     "build": "npm run build:browser && npm run build:main && npm run build:plugins",
     "build-debug": "npm run build:browser-debug && npm run build:plugins",
     "build:browser": "webpack --config browser/webpack.production.config.js",
     "build:browser-debug": "webpack --config browser/webpack.debug.config.js",
     "build:main": "cd main && tsc -p tsconfig.json",
-    "build:plugins": "npm run build:plugin:oni-layer-browser && npm run build:plugin:oni-plugin-typescript && npm run build:plugin:oni-plugin-markdown-preview",
+    "build:plugins":
+      "npm run build:plugin:oni-layer-browser && npm run build:plugin:oni-plugin-typescript && npm run build:plugin:oni-plugin-markdown-preview",
     "build:plugin:oni-layer-browser": "cd extensions/oni-layer-browser && npm run build",
     "build:plugin:oni-plugin-typescript": "cd vim/core/oni-plugin-typescript && npm run build",
-    "build:plugin:oni-plugin-markdown-preview": "cd extensions/oni-plugin-markdown-preview && npm run build",
+    "build:plugin:oni-plugin-markdown-preview":
+      "cd extensions/oni-plugin-markdown-preview && npm run build",
     "build:test": "npm run build:test:unit && npm run build:test:integration",
     "build:test:integration": "cd test && tsc -p tsconfig.json",
     "build:test:unit": "cd browser && tsc -p tsconfig.test.json",
     "copy-icons": "node build/CopyIcons.js",
-    "debug:test:unit:browser": "cd browser && tsc -p tsconfig.test.json && electron-mocha --interactive --debug --renderer --require testHelpers.js -R testCoverageReporter --recursive ../lib_test/browser/test",
+    "debug:test:unit:browser":
```

Comment:
Does prettier make these revisions to the `json` file? I notice that there is some tool (I think its `yarn`) that un-does these edits.

Reply:
@bryphe yes that was `prettier's` doing, it works on `md` and `json` as well an they are currently expanding its support for other languages and filetypes